app/views/editor.ejs

```diff
@@ -33,7 +33,7 @@
                 <a href="#" class="list-group-item list-group-item-action bg-light">Shortcuts</a>
                 <a href="#" class="list-group-item list-group-item-action bg-light">Overview</a>
                 <a href="#" class="list-group-item list-group-item-action bg-light">Events</a>
-                <a href="#" class="list-group-item list-group-item-action bg-light">Profile</a>
+                <a href="/profile" class="list-group-item list-group-item-action bg-light">Profile</a>
```

Comment:
Is this the left side view pane on the editor? It's going to be replaced with a list of files for the current project, file upload, deletion, etc. controls. I just haven't bothered removing the HTML that's there currently. Let's leave this as is for the time being though.